hepcrawl/testlib/fixtures.py

```diff
@@ -65,6 +64,29 @@ def get_node(spider, tag, response=None, text=None, rtype="xml"):
         selector = Selector(response, type=rtype)
     elif text:
         selector = Selector(text=text, type=rtype)
+
     spider._register_namespaces(selector)
     node = selector.xpath(tag)
     return node
+
+
+def get_responses_path(*path_chunks):
+    """
+    :param path_chunks: Optional extra path element to suffix the responses directory with.
+     Example:
+        >> get_responses_path()
+        '/home/myuser/hepcrawl/tests/responses'
+
+        >> get_responses_path('one', 'two')
+        '/home/myuser/hepcrawl/tests/responses/one/two'
```

Comment:
The example should be wrapped in `:` and have a newline right after (I recommend putting it last):
    http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html